app/models/review.rb

```diff
@@ -0,0 +1,10 @@
+class Review < ActiveRecord::Base
+
+  RATING_RANGE = (1..5).to_a
+
+  belongs_to :video
+  belongs_to :user
+
+  validates_presence_of :user_id, :video_id, :rating, :comment
```

Comment:
User and video id's will be set by your app. Having validation on this means you are validating yourself. You should validate only external sources and for internal logic you have tests.